src/pages/add-review-page/add-review-page.tsx

```diff
@@ -83,29 +86,27 @@ function AddReviewPage(): JSX.Element {
       </div>
 
       <div className="add-review">
-        <form action="#" onSubmit={handleSubmit} className="add-review__form">
+        <form action="#" onSubmit={() => handleSubmit} className="add-review__form">
```

Comment:
```suggestion
        <form action="#" onSubmit={handleSubmit} className="add-review__form">
```
А разве есть разница?